### DISCUSSION TITLE
can wiki.js support Admontion in obsidian, which is one enhanced blockquote ?

### DISCUSSION THREAD

**Discussion by zhuzhzh** [requarks/wiki#6415] [Help / Questions] [answered]
Here is the example at https://plugins.javalent.com/admonition/beginner/codeblocks.

https://raw.githubusercontent.com/valentine195/obsidian-admonition/master/images/all.gif

It uses the syntax like codeblocks, and can have some options like title, icon, collapse

```ad-note
title:                  # Admonition title.
collapse:               # Create a collapsible admonition.
icon:                   # Override the icon.
color:                  # Override the color.
<markdown text>
```

```ad-info
<markdown text>
```

I think it's easy to write and more beautiful.

**Comment by NGPixel** [comment, marked as answer]
See the **Stylings** tab in https://docs.requarks.io/editors/markdown#blockquotes

**Reply by zhuzhzh** [reply]
I knew stylings in wiki.js and used it a lot. But I think Admontion in obsidian provides more powerfull feature and easier to write.
